Client/Frontend/Browser/Tab.swift

```diff
@@ -64,6 +64,7 @@ class Tab: NSObject {
     var restoring: Bool = false
     var pendingScreenshot = false
     var url: URL?
+    var contentBlocker: AnyObject?
```

Comment:
I was stumped on this one. I can't do `available` macro on stored properties, yet I need a prop here for ownership of the content blocker. I didn't want a block of setter/getter code wrapped in `available` macros, but I could do that